nixos/modules/services/computing/slurm/slurm.nix

```diff
@@ -429,7 +452,16 @@ in
 
         serviceConfig = {
           Type = "forking";
-          ExecStart = "${wrappedSlurm}/bin/slurmctld";
+          ExecStart =
+            let
+              isLong = optionName: builtins.stringLength optionName > 1;
+              flagFormat = optionName: {
+                option = if isLong optionName then "--${optionName}" else "-${optionName}";
+                sep = " ";
+                explicitBool = false;
+              };
+            in
+            "${wrappedSlurm}/bin/slurmctld ${lib.cli.toCommandLineShell flagFormat cfg.server.flags}";
```

Comment:
`toCommandLineShell` is incorrect here. For `ExecStart`, you should combine `toCommandLine` (or `toCommandLineGNU`) with `utils.escapeSystemdExecArgs`.